### DISCUSSION TITLE
28. Type d'images à utiliser suivant le contexte d'utilisation

### DISCUSSION THREAD

**Discussion by dNicolle** [cnumr/best-practices-wordpress#46] [Bonnes pratiques] [answered]
https://github.com/cnumr/fiches-wordpress-greenit/blob/main/best-practices-WordPress/fiches/28.%20Type%20d'images%20%C3%A0%20utiliser%20suivant%20le%20contexte%20d'utilisation.md

@hrenaud, @florinesueur propose de modifier le titre par : Utiliser le format adéquate pour votre image

**Comment by hrenaud** [comment, marked as answer]
C'est fait
